instances.md

```diff
@@ -2,3 +2,19 @@ Instances
 ========================
 
 Instances are virtual machines that are created from disk images.
+
+To list all instances:
+```
+ops instance list
+```
+
+To create a new instance from an existing image:
+```
+ops instance create -i <name> <image name or ref>
+```
+Each image can only have a single instance running, attempting to create a second instance from an image that already has an instance will fail.
```

Comment:
This is not true, but there is nuance to it.

What you might be experiencing is the same instance on the same port will fail or with the default of 'ops instance create' ran locally writes to the same image file will fail but this can be worked-around. This is definitely not true for cloud deploys

Reply:
awesome, good to know, that's totally what I experienced :) I'll amend the doc.

Reply:
If neither of these are the case and you are still experiencing issues feel free to open up a PR for ops with reproducible steps.

Reply:
updated.